packages/overlay/src/integrations/sentry/components/performance/TransactionDetail.tsx

```diff
@@ -41,14 +43,16 @@ export default function TransactionDetail({ showAll }: { showAll: boolean }) {
       return [];
     }
     const COMPARATORS: Record<TransactionsSortTypes, TransactionsInfoComparator> = {
-      [TRANSACTION_SUMMARY_SORT_KEYS.transaction]: (a, b) => {
-        if (a.event_id! < b.event_id!) return -1;
-        if (a.event_id! > b.event_id!) return 1;
+      [TRANSACTION_SUMMARY_SORT_KEYS.timestamp]: (a, b) => {
+        if (a.start_timestamp < b.start_timestamp) return -1;
+        if (a.start_timestamp > b.start_timestamp) return 1;
```

Comment:
At this point you should be able to do `(a, b) => a.start_timestamp - b.start_timestamp`

Reply:
correct.
 feeling noob 😆